src/bootstrap/bootstrap.py

```diff
@@ -1052,12 +926,10 @@ def bootstrap(help_triggered):
     if not os.path.exists(build.build_dir):
         os.makedirs(build.build_dir)
     lock = acquire_lock(build.build_dir)
-    build.update_submodules()
 
     # Fetch/build the bootstrap
     build.download_toolchain()
     sys.stdout.flush()
-    build.ensure_vendored()
     build.build_bootstrap()
```

Comment:
Love how there's basically nothing left :) I think just the build lock is left and then we can start using rustbuild as the entrypoint